services/worker/.env.example

```
@@ -0,0 +1,2 @@
+DATABASE_URL="postgresql://root@localhost:26257/notifications?sslmode=disable"
+LOG_LEVEL="development"
```

dotenv-linter (4.0.0)

[warning] [QuoteCharacter] The value has quote characters (', ")

(QuoteCharacter)